recipe/meta.yaml

```
@@ -2,48 +2,51 @@
 {% set goname = "k8s.io/" + name %}
 
 {% set version = "3.16.2" %}
-{% set pkg_src = ('src/'+goname).replace("/", "\\" if win else "/") %}
+{% set pkg_src = ('src/'+goname).replace("/",os.sep) %}
```

It looks like `goname` is only used in the `pkg_src` line

Also have found the following trick useful when constructing platform dependent paths 

```suggestion

{% set version = "3.16.2" %}
{% set pkg_src = os.sep.join("src", "k8s.io", name) %}
```